mediacenter/tests/test_utils.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from ..utils import guess_kind
+
+
+@pytest.mark.parametrize('input,expected', [
+    ['pouet.jpg', 'image'],
+    ['pouet.jpeg', 'image'],
+    ['pouet.png', 'image'],
+    ['pouet.mp4', 'video'],
+    ['pouet.avi', 'video'],
+    ['pouet.mp3', 'audio'],
+    ['pouet.ogg', 'audio'],
+    ['pouet.pdf', 'pdf'],
+    ['pouet', None],
+])
```

Comment:
missing a test with an unknown extension (which is different than no extension at all)